### PR TITLE
feat(appengine): Migrate helloworld flex to new path

### PR DESCRIPTION
**Pull request by jahjahbin**
Take 2. Let's do the following:

1) Update existing helloworld Flex sample to use new style for specifying higher versions.
2) Delete preview helloworld sample, not needed anymore.

### CODE REVIEW

File: appengine_flexible/helloworld/app.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Google LLC
```
Comment (tritone) [inline appengine_flexible/helloworld/app.yaml:1]: Copyright dates should not be updated on existing files.

**Review by tritone — APPROVED** [review]

File: appengine_flexible/helloworld/app.yaml (hunk 1 above)
Comment (jahjahbin) [reply appengine_flexible/helloworld/app.yaml]: Undid the copyright updates.

**Comment by jahjahbin** [comment]
We do have another internal change we'd like to make before this goes in -- I'll followup this week.

(it's not the end of the world if this gets merged before then, to be honest...)

**Comment by jahjahbin** [comment]
OK, fast followup :) This is ready to merge. We can remove the label.

**Pull request merged**